prow/plugins/trigger/pull-request_test.go

```diff
@@ -130,16 +131,18 @@ func TestHandlePullRequest(t *testing.T) {
 	var testcases = []struct {
 		name string
 
-		Author        string
-		ShouldBuild   bool
-		ShouldComment bool
-		HasOkToTest   bool
-		prLabel       string
-		prChanges     bool
-		prAction      github.PullRequestEventAction
-		prIsDraft     bool
-		eventSender   string
-		jobToAbort    *prowapi.ProwJob
+		Author             string
+		ShouldBuild        bool
+		ShouldComment      bool
+		HasOkToTest        bool
+		prLabel            string
+		prChanges          bool
+		prAction           github.PullRequestEventAction
+		prIsDraft          bool
+		eventSender        string
+		jobToAbort         *prowapi.ProwJob
+		issueLabelsAdded   []string
+		noIssueLabelsAdded bool
```

Comment:
Please remove this, it is the same as `issueLabelsAdded: nil/empty`

Reply:
Sure, removed that one